lib/MyProfile.dart

```diff
@@ -749,24 +764,184 @@ class MyProfileState extends State {
+                                  } else if (!validPhoneNo) {
+                                    return 'Please enter valid Phone number';
+                                  }
+                                },
+                              )),
+                          Padding(
+                              padding: EdgeInsets.only(
+                                  bottom: 6.0, left: 60, right: 60, top: 10),
+                              child: TextFormField(
+                                onChanged: (value) {
+                                  RegExp exp =
+                                      RegExp(r'(^(?:[+0]9)?[0-9]{10,12}$)');
+                                  if (!exp.hasMatch(value)) {
+                                    setState(() {
+                                      validPhoneNo = false;
+                                    });
+                                  } else {
+                                    setState(() {
+                                      validPhoneNo = true;
+                                    });
+                                  }
+                                },
+                                decoration: new InputDecoration(
+                                    isDense: true,
+                                    contentPadding: new EdgeInsets.symmetric(
+                                        vertical: 10.0, horizontal: 10.0),
+                                    focusedBorder: OutlineInputBorder(
+                                      borderSide: BorderSide(
+                                          color: Colors.black, width: 1.0),
+                                    ),
+                                    enabledBorder: OutlineInputBorder(
+                                      borderSide: BorderSide(
+                                          color: Colors.black, width: 1.0),
+                                    ),
+                                    hintText: 'Phone Number-2',
+                                    labelText: "Phone Number-2"),
+                              )),
+                          Padding(
+                              padding: EdgeInsets.only(
+                                  bottom: 6.0, left: 60, right: 60, top: 10),
+                              child: TextFormField(
+                                decoration: new InputDecoration(
+                                    isDense: true,
+                                    contentPadding: new EdgeInsets.symmetric(
+                                        vertical: 10.0, horizontal: 10.0),
+                                    focusedBorder: OutlineInputBorder(
+                                      borderSide: BorderSide(
+                                          color: Colors.black, width: 1.0),
+                                    ),
+                                    enabledBorder: OutlineInputBorder(
+                                      borderSide: BorderSide(
```

Comment:
add email validator similar to login page

Reply:
done